src/services/login-service.js

```diff
@@ -1,35 +1,31 @@
 const jwt = require('jsonwebtoken');
 const bcrypt = require('bcryptjs');
-const userData = require('../database/user-data');
+const userRepository = require('../database/user-repository');
 const SECRET_KEY = process.env.SECRET_KEY;
 
 const createLog = async ({ mail, password }) => {
     try {
-        const user = await userData.getMail(mail);
-        if(user[0].length === 0){
+        const user = await userRepository.findUserByEmail(mail);
+        if(user.length === 0){
             throw{
-                status: 400,
-                message: "Mail not found"
+                status: 404,
+                message: "Mail or password wrong"
             };
         } else {
-            const userCreden = user[0][0];
-            const validatePass = bcrypt.compareSync(password, userCreden.password);
-            if(validatePass){
-                const expiresIn = '10h';
-                const accessToken = jwt.sign({ id: userCreden.user }, SECRET_KEY, { expiresIn: expiresIn });
+            const userCreden = user[0];
```

Comment:
this should be in the repository

or can gime me an example of the response pliss?